flytekit/core/python_function_task.py

```diff
@@ -15,7 +15,8 @@
 from flytekit.models import literals as _literal_models
 from flytekit.models import task as _task_model
 
-_IMAGE_REPLACE_REGEX = re.compile(r"({{\s*.image[s]?.(\w+).(\w+)\s*}})", re.IGNORECASE)
+
+_IMAGE_REPLACE_REGEX = re.compile(r"({{\s*\.image[s]?(?:\.([a-zA-Z]+))(?:\.([a-zA-Z]+))?\s*}})", re.IGNORECASE)
```

Comment:
could we maybe document what this regex matches?

Reply:
Added... one more look?